sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -22,17 +50,224 @@ class TableCheckpointStore:
+            u'owner_id' : ownership['owner_id'],
+        }
+        return ownership_entity
+
+    @classmethod
+    def _create_checkpoint_entity(cls, checkpoint):
+        """
+        create a dictionary with the new checkpoint attributes so that it can be updated in tables
+        """
+        checkpoint_entity = {
+            u'PartitionKey': "{} {} {} Checkpoint".format(checkpoint["fully_qualified_namespace"],
+        checkpoint["eventhub_name"], checkpoint['consumer_group']),
+            u'RowKey': checkpoint['partition_id'],
+            u'offset' : checkpoint['offset'],
+            u'sequence_number' : checkpoint['sequence_number'],}
+        return checkpoint_entity
+
+    def list_ownership(self, fully_qualified_namespace, eventhub_name, consumer_group):
+        """Retrieves a complete ownership list from the storage table.
+        :param str fully_qualified_namespace: The fully qualified namespace that the Event Hub belongs to.
+         The format is like "<namespace>.servicebus.windows.net".
+        :param str eventhub_name: The name of the specific Event Hub the partition ownerships are associated with,
+         relative to the Event Hubs namespace that contains it.
+        :param str consumer_group: The name of the consumer group the ownerships are associated with.
+        :rtype: Iterable[Dict[str, Any]], Iterable of dictionaries containing partition ownership information:
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
+                  The format is like "<namespace>.servicebus.windows.net".
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the ownership are associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `owner_id` (str): A UUID representing the current owner of this partition.
+                - `last_modified_time` (float): The last time this ownership was claimed.
+                - `etag` (str): The Etag value for the last time this ownership was modified. Optional depending
+                  on storage implementation.
+        """
+        partition_key = "{} {} {} Ownership".format(fully_qualified_namespace,
+        eventhub_name, consumer_group)
+        my_filter = "PartitionKey eq '"+ partition_key + "'"
+        entities = self.table_client.query_entities(my_filter)
+        ownershiplist = []
+        for entity in entities:
+            dic = {}
+            dic[u'fully_qualified_namespace'] = fully_qualified_namespace
+            dic[u'eventhub_name'] = eventhub_name
+            dic[u'consumer_group'] = consumer_group
+            dic[u'partition_id'] = entity[u'RowKey']
+            dic[u'owner_id'] = entity[u'owner_id']
```

Comment:
The owner_id when stored in the entity should be `ownerid`:
```suggestion
            dic[u'owner_id'] = entity[u'ownerid']
```